Makefile

```diff
@@ -0,0 +1,72 @@
+# -- Docker
+UID              = $(shell id -u)
+COMPOSE          = docker-compose
+COMPOSE_RUN      = $(COMPOSE) run --rm
+COMPOSE_RUN_APP  = $(COMPOSE_RUN) app
+COMPOSE_EXEC     = $(COMPOSE) exec
+COMPOSE_EXEC_APP = $(COMPOSE_EXEC) app
+
+# -- Django
+MANAGE = $(COMPOSE_RUN_APP) dockerize -wait tcp://db:5432 -timeout 60s \
+	python manage.py
+
+# -- Rules
+default: help
+
+bootstrap: data/media/.keep data/static/.keep build run migrate  ## install development dependencies
+.PHONY: bootstrap
+
+# == Docker
+build: ## build the app container
+	@$(COMPOSE) build --build-arg UID=$(UID) app
+.PHONY: build
+
+down: ## stop & remove containers
+	@$(COMPOSE) down
+.PHONY: down
+
+logs: ## display app logs (follow mode)
+	@$(COMPOSE) logs -f app
+.PHONY: logs
+
+run: ## start the development server
+	@$(COMPOSE) up -d
+.PHONY: run
+
+stop: ## stop the development server
+	@$(COMPOSE) stop
+.PHONY: stop
+
+# == Django tasks
+demo-site:  ## create a demo site
+	@$(MANAGE) create_demo_site
+	@${MAKE} search-index;
+.PHONY: demo-site
+
+migrate:  ## perform database migrations
+	@$(MANAGE) migrate
+.PHONY: migrate
+
+search-index:  ## (re)generate the Elasticsearch index
+	@$(MANAGE) bootstrap_elasticsearch
+.PHONY: search-index
+
+superuser: ## create a DjangoCMS superuser
+	@$(MANAGE) createsuperuser
+.PHONY: superuser
+
+# == Misc
+clean: ## restore repository state as it was freshly cloned
```

Comment:
`.PHONY` is missing for this task please.

Reply:
Well spotted!